contrib/vagrant/control-plane-base.sh

```diff
@@ -1,6 +1,161 @@
 #!/bin/bash
 
-zypper install -y -t pattern kubic_admin
-install -D -m 0644 /home/vagrant/enclave/contrib/crio/00-default.conf /etc/crio/crio.conf.d/00-default.conf
-systemctl restart crio
-systemctl enable kubelet.service
+# # zypper install -y -t pattern kubic_admin
```

Comment:
I'm starting to wonder if the Vagrant bash files should be made generic and in a separate directory. Perhaps with the config files split as well instead of being embedded in the scripts themselves. Thoughts?

Reply:
I can try moving out the config files, good idea.

re: keeping them in a separate directory - isn't `contrib/vagrant` enough?

Reply:
> isn't `contrib/vagrant` enough?

Probably! I wanted to at least introduce the idea if we wanted to make the convieniece scripts generalized and not just for vagrant.

Reply:
Good idea. I will name it `contrib/provision` then.